src/frontend/src/pages/ProjectDetailPage/ProjectEdit/ProjectEditDetails.tsx

```diff
@@ -33,9 +34,12 @@ const ProjectEditDetails: React.FC<ProjectEditDetailsProps> = ({
   setProjectManager
 }) => {
   return (
-    <PageBlock title="Project Details">
-      <Grid container spacing={2}>
-        <Grid item xs={8}>
+    <Box>
+      <Typography variant="h5" sx={{ marginBottom: '10px' }}>
+        Project Details
+      </Typography>
+      <Grid container spacing={3}>
+        <Grid item lg={2.4} xs={12}>
```

Comment:
you should use an md between lg and xs, it switches to the stacked view too quickly rn